Directory.Packages.props

```diff
@@ -2,38 +2,35 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    <AspireVersion>9.0.0-rc.1.24511.1</AspireVersion>
+    <AspireVersion>9.0.0</AspireVersion>
   </PropertyGroup>
-
   <ItemGroup>
     <PackageVersion Include="Aspire.Hosting" Version="$(AspireVersion)" />
     <PackageVersion Include="Aspire.Hosting.AppHost" Version="$(AspireVersion)" />
+    <PackageVersion Include="Aspire.Hosting.Testing" Version="$(AspireVersion)" />	
     <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery.Dns" Version="$(AspireVersion)" />
-
-    <PackageVersion Include="Microsoft.Extensions.Http.Resilience" Version="9.0.0-preview.9.24507.7" />
-
+    <PackageVersion Include="Microsoft.Extensions.Http.Resilience" Version="9.0.0" />
```

Comment:
nit: why not use $(AspireVersion) here as well.

Reply:
Fixed after rebasing from `dev`.